src/web/components/Core/Collapsible.tsx

```diff
@@ -8,21 +8,25 @@ export type CollapsibleProps = {
   title: string;
   content: ReactNode;
   defaultOpen?: boolean;
+  label?: string;
 };
 
-export function Collapsible({ title, content, defaultOpen }: CollapsibleProps) {
+export function Collapsible({ title, content, defaultOpen, label }: CollapsibleProps) {
   return (
     <RadixCollapsible.Root className='collapsible-root' defaultOpen={defaultOpen}>
-      <div className='collapsible-header'>
-        <h2>{title}</h2>
-        <RadixCollapsible.Trigger className='collapsible-trigger'>
-          <FontAwesomeIcon
-            icon='chevron-down'
-            data-testid='chevron-icon'
-            className='chevron-icon'
-          />
-        </RadixCollapsible.Trigger>
-      </div>
+      <RadixCollapsible.Trigger className='collapsible-trigger'>
+        <div className='collapsible-header'>
+          <h2>{title}</h2>
```

Comment:
How about taking a header prop which is a ReactNode? So that if we need to make the component looks different when collapsed we could just pass into this prop?

Reply:
ah good idea. to clarify, do you propose that the title and the icon would be part of this header too?

Reply:
Yea, I am thinking something like
```
<Collapsible header={
 <div className='collapsible-header'>
      <h2>{title}</h2>
      <div className='collapsible-header-label-and-icon'>
        {label && <span className='label'>{label}</span>}
        <FontAwesomeIcon
          icon='chevron-down'
          data-testid='chevron-icon'
          className='chevron-icon'
        />
      </div>
    </div>
}/>
```

Just for discussion if you think this makes sense?

Reply:
definitely a discussion worth having. I would have thought that all the uses of the collapsible would use the same h2 for the title, and the same chevron for the icon. So the text for the title as well as the label are the only things that would differ, which would lead more to just having title and label props, so that the consumer doesn't have to repeat that pattern on every usage.

One alternative could be for the label prop to be a ReactNode? Though I would've thought only the styling (e.g. background colour) might change, so perhaps not the best idea.

Does that sound reasonable?

Reply:
I see, yes that's reasonable to me. Let's just leave label in this component

Reply:
great, thanks for advising!